scripts/pylint_extensions.py

```diff
@@ -1228,40 +1228,50 @@ def process_module(self, node):
             if line_num > 0:
                 previous_line = file_content[line_num - 1].strip()
 
+            # Comments may include a lowercase character at beginning
+            # or may not use a punctuation at end if it contains a
+            # excluded phrase e.g. "# coding: utf-8".
             if line.startswith(b'#'):
                 # Check if comment contains any excluded phrase.
                 word_is_present_in_excluded_phrases = any(
                     word in line for word in EXCLUDED_PHRASES)
 
-                # Check if variable name is used.
-                underscore_is_present = any('_' in word for word in line)
-
-                if word_is_present_in_excluded_phrases or underscore_is_present:
+                if word_is_present_in_excluded_phrases:
                     continue
 
             if line.startswith(b'#') and not next_line.startswith(b'#'):
```

Comment:
I notice the "if line.startswith(b'#'):" construct in multiple places. Would it be simpler to just have a "if not line.startswith(b'#'): continue" at the beginning, and then we don't have to keep checking this in every stanza?

Reply:
Done.